docs/aws/README.md

```diff
@@ -3,31 +3,32 @@
 This guide will provide a complete Karpenter installation for AWS.
 These steps are opinionated and may need to be adapted for your use case.
 
+> This guide should take less than 1 hour to complete and should cost less than $.25
```

Comment:
ultra minor nit, but you've stated should twice. Transitive property :)
`This guide should take less than 1 hour to complete and cost less than $.25`